src/utils/LockOverviewUtils.ts

```diff
@@ -96,3 +97,32 @@ export const calculateReturn = (
 export const calculateGasBuffer = (gasFee: number) => {
   return gasFee + gasFee * 0.1
 }
+
+export const convertBigIntToNumber = (
```

Comment:
there is already a BigNumber.toNumber( ) function and this one converts to Float (which might apply an unexpected rounding)

Reply:
https://docs.ethers.io/v5/api/utils/bignumber/#:~:text=which%20support%20them.-,BigNumber,-.toNumber(

I was getting this error when I tried it.

Reply:
if you get a min/max error it means that code is wrong somewhere, hiding that error using another function that does parsing and rounding its not a proper solution

Reply:
I understand better now. Let me implement that flow now.